tests/h/services/document_test.py

```diff
@@ -87,40 +89,46 @@ def test_it_returns_documents_ordered_by_last_activity_desc(
             annotations[0].document,
         ]
 
+    @pytest.fixture
+    def target_user(self, factories):
+        return factories.User()
 
-@pytest.fixture
-def target_user(factories):
-    return factories.User()
-
-
-@pytest.fixture
-def other_user(factories):
-    return factories.User()
+    @pytest.fixture
+    def other_user(self, factories):
+        return factories.User()
 
+    @pytest.fixture
+    def groups(self, factories):
+        return {"target_group": factories.Group(), "other_group": factories.Group()}
 
-@pytest.fixture
-def groups(factories):
-    return {"target_group": factories.Group(), "other_group": factories.Group()}
+    @pytest.fixture
+    def annotations(self, factories, groups):
+        return [
+            factories.Annotation(groupid=groups["target_group"].pubid, shared=True),
+            factories.Annotation(groupid=groups["target_group"].pubid, shared=True),
+            factories.Annotation(groupid=groups["target_group"].pubid, shared=True),
+        ]
 
+    @pytest.fixture
+    def other_annotations(self, factories, groups):
+        return [
+            factories.Annotation(groupid=groups["other_group"].pubid, shared=True),
+            factories.Annotation(groupid=groups["other_group"].pubid, shared=True),
+            factories.Annotation(groupid=groups["other_group"].pubid, shared=True),
+        ]
 
-@pytest.fixture
-def annotations(factories, groups):
-    return [
-        factories.Annotation(groupid=groups["target_group"].pubid, shared=True),
-        factories.Annotation(groupid=groups["target_group"].pubid, shared=True),
-        factories.Annotation(groupid=groups["target_group"].pubid, shared=True),
-    ]
+    @pytest.fixture
+    def svc(self, db_session):
+        return DocumentService(session=db_session)
 
 
-@pytest.fixture
-def other_annotations(factories, groups):
-    return [
-        factories.Annotation(groupid=groups["other_group"].pubid, shared=True),
-        factories.Annotation(groupid=groups["other_group"].pubid, shared=True),
-        factories.Annotation(groupid=groups["other_group"].pubid, shared=True),
-    ]
```

Comment:
The changes above here are all indenting the methods to attach them to the relevant class instead of having them floating about at the bottom. This leaves space for our service factory test.